static/app/types/event.tsx

```diff
@@ -47,12 +47,18 @@ export type VariantEvidence = {
   parent_span_ids?: string[];
 };
 
-type EventGroupVariantKey = 'custom-fingerprint' | 'app' | 'default' | 'system';
+type EventGroupVariantKey =
+  | 'built-in-fingerprint'
+  | 'custom-fingerprint'
+  | 'app'
+  | 'default'
+  | 'system';
 
 export enum EventGroupVariantType {
   CHECKSUM = 'checksum',
   FALLBACK = 'fallback',
   CUSTOM_FINGERPRINT = 'custom-fingerprint',
+  BUILT_IN_FINGERPRINT = 'built-in-fingerprint',
   COMPONENT = 'component',
   SALTED_COMPONENT = 'salted-component',
```

Comment:
Out of curiosity, what are salted components?

Reply:
Peanuts, pretzels, and ...

https://github.com/getsentry/sentry/blob/88850793e9c962dbba01d77d335b27ebf41fb224/src/sentry/grouping/__init__.py#L59-L63